controllers/validator_test.go

```diff
@@ -1038,14 +1038,17 @@ func TestDPAReconciler_ValidateDataProtectionCR(t *testing.T) {
 					SnapshotLocations: []oadpv1alpha1.SnapshotLocation{
 						{
 							Velero: &v1.VolumeSnapshotLocationSpec{
-								Provider: "velero.io/aws",
```

Comment:
Looking this code, I think this is not a valid change
https://github.com/openshift/oadp-operator/blob/fb2411e2948874a6abe61ca31b1cd9957621ebb6/controllers/bsl.go#L50-L68
validation should accept both `aws` and `velero.io/aws` (but this should be fixed in the future, we can open an issue to discuss only allowing `aws`), right maintainers?

Reply:
@mateusoliveira43 I have fixed this intentionally, for now , for this test to pass.

Because this is a bug here actually, there are multiple issues regarding this, which i plan to open soon.

Then, we can separately fix this issue.

Reply:
Since this changes behavior (prior to this PR, users could specify `aws` or `velero.io/aws` in VSL, and after it, users will be able to only specify `aws`) I think is important to discuss this prior to merge

(easy solution could be that if user specifies `velero.io/aws`, the error message tell it to use `aws` instead, but we need to verity our deprecation policy as well, maybe first we need to deprecate it)

Reply:
no, no, this PR does not change any behaviour, i have only modified the test case to use "aws" for sometime.
Earlier, velero.io/aws was also creating a Warning event, and was getting ignored.

The reason it is failing now with "velero.io" prefix is a different bug, which is causing issue now. I will raise a thread in slack. That needs to be fixed first, and after that, we can have any value here.